x/launch/keeper/msg_server_request_add_account.go

```diff
@@ -19,6 +19,10 @@ func (k msgServer) RequestAddAccount(
 		return nil, sdkerrors.Wrapf(types.ErrChainNotFound, "%v", msg.ChainID)
 	}
 
+	if chain.IsMainnet {
+		return nil, sdkerrors.Wrapf(types.ErrChainIsMainnet, "%v", msg.ChainID)
```

Comment:
I recommend not using %v if you we know the type of the argument for better type safety.

Reply:
Good point

I added it here https://github.com/tendermint/spn/issues/299 to do it everywhere